Library/Homebrew/cli/named_args.rb

```diff
@@ -246,30 +286,33 @@ def load_formula_or_cask(name, only: nil, method: nil, warn: nil)
 
           user, repo, short_name = name.downcase.split("/", 3)
           if repo.present? && short_name.present?
-            tap = Tap.fetch(user, repo)
+            tap = Tap.fetch(T.must(user), repo)
             raise TapFormulaOrCaskUnavailableError.new(tap, short_name)
           end
 
           raise NoSuchKegError, name if resolve_formula(name)
-
-          raise FormulaOrCaskUnavailableError, name
```

Comment:
We can't reach this, bc `resolve_formula` either returns a Formula or raises (e.g. `FormulaUnavailableError`)